leasing/management/commands/attach_areas.py

```diff
@@ -22,28 +24,37 @@ class Command(BaseCommand):
     def handle(self, *args, **options):  # noqa: C901 TODO
         from auditlog.registry import auditlog
 
+        logging.basicConfig(level=logging.INFO, format="%(message)s", stream=sys.stdout)
+        log = logging.getLogger(__name__)
+
+        verbosity = options.get("verbosity")
+        if verbosity == 0:
+            log.setLevel(logging.WARNING)
+        elif verbosity == 2:
+            log.setLevel(logging.DEBUG)
+
         # Unregister all models from auditlog when importing
         for model in list(auditlog._registry.keys()):
             auditlog.unregister(model)
 
         leases = Lease.objects.all()
 
+        log.info(f"Processing {leases.count()} objects.")
+
         for lease in leases:
-            self.stdout.write("Lease #{} {}:".format(lease.id, lease.identifier))
+            log.debug("Lease #{} {}:".format(lease.id, lease.identifier))
```

Comment:
It would be better to convert the format strings to the "standard" format used by the Python logger so that you could utilize the lazy formatting. I.e. in this case:
```python
LOG.debug("Lease #%s %s:", lease.id, lease.identifier)
```
That would have the following benefits:
1. If the log message is filtered by the log level configuration, the string with the placeholders replaced won't even be generated and some processing time is saved.
2. If the log messages are directed some other tooling such as Sentry, then Sentry can group those messages based on the first parameter, i.e. it knows that they're instance of the same event even though the ids and identifiers are different.

Reply:
Here's a link for more info about the redirection of logging to Sentry, since I think it's pretty useful feature that could be utilized more often and therefore I want to increase knowledge about it. https://docs.sentry.io/platforms/python/guides/logging/

Reply:
Converted